js/src/edit-block/components/settings/email.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Input } from '../';
+
+/**
+ * @typedef {Object} EmailProps The component props.
+ * @property {Object} setting This setting.
+ * @property {string|undefined} value The setting value.
+ * @property {Function} handleOnChange Handles a change in this setting.
+ */
+
+/**
+ * The number component.
+ *
+ * @param {EmailProps} props The component props.
+ * @return {React.ReactElement} The number component.
+ */
+const Email = ( props ) => {
+	return <Input { ...props } type="email" />;
+};
```

Comment:
This adds a default setting to the Email field. I forgot this before.

![email-field-default](https://user-images.githubusercontent.com/4063887/105670050-0405c080-5ea6-11eb-9504-99fdb53698cd.png)